### ISSUE TITLE
Formatting of online documentation home page for 4.2.2 is a bit compact.

### ISSUE THREAD

**Issue by reticulatus** [wxWidgets/Phoenix#2608]
Looking at https://docs.wxpython.org/ the layout of the home page looks a bit scrunched up compared to 4.2.1.

![Docs_4 2 2](https://github.com/user-attachments/assets/b2ba9e20-f26a-448b-b3bd-479dd4323b7c)

![Docs_4 2 1](https://github.com/user-attachments/assets/45547023-9ed8-4119-991a-59b4630ef09e)

I get similar results in Firefox, Waterfox and Chromium browsers.

**Comment by swt2c** [comment]
Sigh, yeah I ran into lots of issues trying to build the documentation - mostly related to sphinx dependencies that have been updated but no longer work with old sphinx (2.2.0).  The first issue you noted seems to be a bug in the generation.

I might just try updating to the latest sphinx (and unpin all its dependencies) and see how things look.

**Comment by reticulatus** [comment]
I see that there are non-dev `tar.gz` files of the demo (wxPython-demo-4.2.2.tar.gz), docs (wxPython-docs-4.2.2.tar.gz) and docset (wxPython-docset-4.2.2.tar.gz) in https://wxpython.org/Phoenix/snapshot-builds/

Should they be in https://extras.wxpython.org/wxPython4/extras/4.2.2/ ?

**Comment by swt2c** [comment]
> I see that there are non-dev `tar.gz` files of the demo (wxPython-demo-4.2.2.tar.gz), docs (wxPython-docs-4.2.2.tar.gz) and docset (wxPython-docset-4.2.2.tar.gz) in https://wxpython.org/Phoenix/snapshot-builds/
> 
> Should they be in a new folder: https://extras.wxpython.org/wxPython4/extras/4.2.2/ ?

Yes, I fixed that manually for now.  I didn't automate that part yet.  I also am not sure about those pdb files.

**Comment by reticulatus** [comment]
I noticed in line 95 of the `index.html` file in 4.2.2, there is an `</aside>` tag. When I replaced that tag with `</div>` (like there was in the 4.2.1 version), the formatting of the page appeared to be fixed. 

It might be worth considering this as a temporary fix while you investigate the issue.

**Comment by swt2c** [comment]
Thanks!  I made that change by hand.  That definitely seems to have been the bug caused by <something>.

Meanwhile, that bug doesn't exist when I build with the latest sphinx (but others do, sigh) so...
